### DISCUSSION TITLE
How to go about CDN approach for API response?

### DISCUSSION THREAD

**Discussion by GorvGoyl** [vercel/next.js#23816] [Help]
Suppose, From my react component I call a NextJS api route to fetch data. This API route calls an external API to fetch the data and returns back to component. This fetched data doesn't changed frequently (once a month).

Now, can I do some CDN approach for this API route where it instantly returns the cached data without calling the external api?

I know I could do 'cache-control' inside request header but that'd cache result only for that single browser not globally. I want to cache result globally.

**Comment by akellbl4** [comment]
Hi, I understand your problem. Yes, you can do so.
But then you need to think about where you will store your cached data and etc. These questions NodeJS and backend-specific and because Next.js doesn't have such functionality.

**Reply by GorvGoyl** [reply]
makes sense, thanks.
